pkg/controller.v1/pytorch/envvar.go

```diff
@@ -22,13 +22,40 @@ import (
 	commonv1 "github.com/kubeflow/common/pkg/apis/common/v1"
 	pytorchv1 "github.com/kubeflow/training-operator/pkg/apis/pytorch/v1"
 	corev1 "k8s.io/api/core/v1"
+	volcanov1beta1 "volcano.sh/apis/pkg/apis/scheduling/v1beta1"
 )
 
 // EnvVarGenerator is the environment variable generator interface.
 type EnvVarGenerator interface {
 	Generate(job *pytorchv1.PyTorchJob) ([]corev1.EnvVar, error)
 }
 
+func setPodLabel(obj interface{}, podTemplateSpec *corev1.PodTemplateSpec, rtype, index string) error {
```

Comment:
0. This function name seems not self-explanatory enough. Without looking into the function body, reader may never know this function actually set the volcano preemptable labels for **PodTemplate** in PyTorchJob.
1. I do Not support adding volcano labels directly into the spec of PyTorchJobs

Reply:
I just want to  reserve an extension here by the way. If there is a need for adding a new label in the future, we can write it here.